kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/NonJVMClientsST.java

```diff
@@ -51,6 +52,8 @@ class NonJVMClientsST extends AbstractST {
      */
     @Test
     void produceAndConsumeWithKcatClients(String namespace) {
+        // Skip Kcat execution when architecture is different from x86_64
+        assumeThat(Environment.ARCHITECTURE.equals(Environment.ARCHITECTURE_DEFAULT)).isTrue();
```

Comment:
Let's use the `os.arch` java system property.  assumeThat it is x86 (you'll need to check how it is formulated) and skip otherwise.